### PR TITLE
Catalog, Search: add help dropdown to the index landing page

### PR DESCRIPTION
**Pull request by fiskus**
Search Help has a slightly different look for Header and OpenLanding. I made `NavBar/Help.js` more abstract (in terms of styles) and reused for OpenLanding. Maybe I should move it to another place, ex. `components/SearchHelp.js`

demo

### CODE REVIEW

File: catalog/app/containers/NavBar/Help.js [hunk 1]
```diff
@@ -247,11 +238,17 @@ function DocsExternalLink() {
   )
 }
 
-export default function Help({ className, onQuery, ...props }) {
+function normalizeSyntaxItem(s) {
+  const multilineToLine = s.replace(/\s/g, '')
+  const trailingSpace = ' '
```
Comment (nl0) [inline catalog/app/containers/NavBar/Help.js:243]: i dont think it makes sense to instantiate this variable every time the function gets called, just `return "{mulitilineToLine} "` (wrong quotes bc of markdown syntax) or even make it into a oneliner

**Review by nl0 — CHANGES_REQUESTED** [review]
The changes look good overall, but i think the huge rounded corners look a little out-of-place. i'd rather go for the standard border-radius, like this:
localhost_3000_(Laptop 2x)

File: catalog/app/containers/NavBar/Help.js (hunk 1 above)
Comment (fiskus) [reply catalog/app/containers/NavBar/Help.js:243]: The performance cost is insignificant here in my opinion, the transpiller even can remove these variables. But I think it's simpler to read code when all things have some explanations through variable names.

Comment (fiskus) [reply catalog/app/containers/NavBar/Help.js:243]: For example, in the future when I will see `{multiline} ` I can think that this extra space is a mistyping

Comment (fiskus) [reply catalog/app/containers/NavBar/Help.js]: Converted to oneliner

**Comment by fiskus** [comment]
> the huge rounded corners look a little out-of-place. i'd rather go for the standard border-radius

Removed too much rounded corners as you propose

**Review by nl0 — APPROVED** [review]
haven't tested it after the most recent changes, but i believe it should be ok

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
 ## Catalog, Lambdas
 * [Added] Retry logic for failed queries, minimize load on ES for sample, images
+* [Added] Search help dropdown for the index landing page ([1838](https://github.com/quiltdata/quilt/pull/1838))
```
Comment (nl0) [inline docs/CHANGELOG.md:17]: i believe the pr number should be prefixed with a `#` by convention

Comment (fiskus) [reply docs/CHANGELOG.md:17]: Thanks!

**Pull request merged**